src/brian.py

```diff
@@ -156,7 +156,7 @@ def start_new():
             if want_to_start and good_to_start and dsp_finish_counter:
                 # Acknowledge new sequence can begin to Radar Control by requesting new sequence metadata
                 log.debug("requesting metadata from radar_control")
-                so.send_request(
+                so.send_data(
```

Comment:
I seem to recall discussing possibly renaming this function to `send_string`?

Reply:
Indeed we did. That must of slipped off the plate of things to do while I was working on the sockets.